### ISSUE TITLE
Record rank history and/or upload custom datapoints?

### ISSUE THREAD

**Issue by NafeeJ** [Adrriii/osudaily#4]
Wondering if recording the past 90 days of rank history, that osu! shows by default, with null values for the other stuff (playcount, pp, etc.) is possible so that data isn't lost while its still there

Also, if a user has previous profile screenshots that they wish to manually enter as datapoints themselves?

**Comment by Adrriii** [comment]
Hey, the only correct value would be the rank here, even performance wouldn't be right cause it's an "approximation" (the closest player on the site with that rank)

So it's not really usefull cause it would be missing literally anything else, and also the plot would be very hard to parse in the first place.

About screenshots, i'm not willing to go through them manually but nothing prevents you from fetching your stats in csv and do as you need with them and add your own stats :)
